release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -521,6 +521,70 @@ If the following conditions are true, the feature is enabled by default:
 [id="ocp-4-15-monitoring"]
 === Monitoring
 
+The in-cluster monitoring stack for this release includes the following new and modified features.
+
+[id="ocp-4-15-monitoring-updates-to-monitoring-stack-components-and-dependencies"]
+==== Updates to monitoring stack components and dependencies
+This release includes the following version updates for in-cluster monitoring stack components and dependencies:
+
+* Alertmanager to 0.26.0
+* kube-state-metrics to 2.10.1
+* node-exporter to 1.7.0
+* Prometheus to 2.48.0
+* Prometheus Adapter to 0.11.2
+* Prometheus Operator to 0.70.0
+* Thanos Querier to 0.32.5
+
+[id="ocp-4-15-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red{nbsp}Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* The `NodeClockNotSynchronising` and `NodeClockSkewDetected` alerting rules are now disabled when the Precision Time Protocol (PTP) is in use.
+
+[id="ocp-4-15-monitoring-new-metrics-server-to-access-metrics-API-technology-preview"]
+==== New Metrics Server component to access the Metrics API (Technology Preview)
+This release introduces a Technology Preview option to add a Metrics Server component to the in-cluster monitoring stack.
+As a Technology Preview feature, Metrics Server is automatically installed instead of Prometheus Adapter if the `FeatureGate` custom resource is configured with the `TechPreviewNoUpgrade` option.
+If installed, Metrics Server, rather than Prometheus Adapter, collects resource metrics and exposes them in the `metrics.k8s.io` Metrics API service for use by other tools and APIs.
```

Comment:
(nit) it reads as if Prometheus adapter collects metrics too which isn't the case.
```suggestion
If installed, Metrics Server collects resource metrics and exposes them in the `metrics.k8s.io` Metrics API service for use by other tools and APIs.
```

Reply:
Removed references to Prometheus Adapter